src/components/CreateArticleForm.jsx

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import {
+  Button,
+  Container,
+  Form,
+  Header,
+  Input,
+  TextArea,
+  Message,
+  } from "semantic-ui-react";
+
+import { useDispatch, useSelector } from "react-redux";
+import { CreateArticle } from "../modules/CreateArticle";
+
+function CreateArticleForm() {
+  const dispatch = useDispatch();
+  const { createArticleMessage, errorMessage } = useSelector(state => state);
+
+  return (
+    <Container>
+      <Header>Create Article</Header>
+      <Form
+        data-cy="article-form"
+        onSubmit={(event) => CreateArticle.create(event, dispatch)}
+      >
+        <Form.Field
+          data-cy="title-field"
+          label="Article title"
+          control={Input}
+          name="title"
+          placeholder="Title"
+        />
+        <Form.Field
+          data-cy="lead-field"
+          label="Article lead"
+          control={Input}
+          name="lead"
+          placeholder="Lead"
+        />
+        <Form.Field
+          data-cy="body-field"
+          label="Article body"
+          control={TextArea}
+          name="body"
+          placeholder="Body"
+        />
+        <Form.Field>
+        <label for="categories">Choose a category:</label>
+        <select name="categories" id="categories" data-cy="categories-dropdown">
```

Comment:
why are you using a `select` here instead of a semantic ui element?

Reply:
To find the target.value for the dropdown and to make the cypress tests go green.
 We have implemented Semantic via putting it in a ```<Form.Field>```